packages/disputer/src/disputer.js

```diff
@@ -231,9 +231,13 @@ class Disputer {
       : this.account;
 
     // Can only derive rewards from disputed liquidations that this account disputed.
-    const disputedLiquidations = this.financialContractClient
-      .getDisputedLiquidations()
-      .filter((liquidation) => liquidation.disputer === disputerAddress);
+    const disputedLiquidations = this.financialContractClient.getDisputedLiquidations().filter((liquidation) => {
```

Comment:
we can clean up this code, by applying filter only when WITHDRAW_ALL_LIQUIDATIONS is not there.

Reply:
Also, can you add a logger statement, so that we know, if liquidations have been filtered or not

Reply:
simplified it a bit